docs/content/en/4.providers/twicpics.md

```diff
@@ -5,16 +5,437 @@ navigation:
+
+Syntax: `{ focus: `[`coordinates`](https://www.twicpics.com/docs/api/manipulations/?utm_source=nuxt&utm_medium=organic&utm_campaign=provider#crop-coordinates)`|'auto' }`
+
+```html
+<NuxtImg
+  provider="twicpics"
+  src="/cat_1x1.jpg"
+  :modifiers="{focus:'auto', crop:'500x500'}" <!-- using crop with focus auto -->
+/>
+
+<NuxtImg
+  provider="twicpics"
+  src="/football.jpg"
+  :modifiers="{focus:'auto', cover:'1:1'}" <!-- changing ratio with focus auto -->
+/>
+
+<NuxtImg
+  provider="twicpics"
+  src="/path-to-nuxt-demo.jpg"
+  :modifiers="{focus:'200x200', cover:'1:1'}" <!-- changing ratio with selected focus -->
+/>
+```
+
+_NB_ : focus must be placed before the transformations modifying the output image (namely cover, crop and resize).
+
+More informations [about `focus` here](https://www.twicpics.com/docs/api/transformations/?utm_source=nuxt&utm_medium=organic&utm_campaign=provider#focus).
+
+### truecolor
+
+`truecolor` can be used to prevent color quantization. If no boolean is provided, true is assumed.
+
+By default, quantization is allowed (truecolor=false).
+
+Quantization occurs whenever the output format is `png`.
+
+Use truecolor if you want to distribute substantially larger but more accurate images with translucency to users on browsers with no `WebP` support or when `png` is required as output format.
+
+```html
+<NuxtImg
+  provider="twicpics"
+  src="/components/peacock.jpg"
+  format="png" 
+  :modifiers="{truecolor:true}" <!-- disallowes color quantization -->
+/>
+
+<NuxtImg
+  provider="twicpics"
+  src="/components/peacock.jpg"
+  format="png"
+  :modifiers="{truecolor:false}" <!-- allowes color quantization (default value) -->
+/>
+```
+
+More informations [about `truecolor` here](https://www.twicpics.com/docs/api/transformations/?utm_source=nuxt&utm_medium=organic&utm_campaign=provider#truecolor).
+
+### turn
+
+`turn` will change the orientation of the image. It accepts an angle in degrees or an expression. Angles will be rounded to the closest multiple of 90°.
+
+Syntax: `{ turn: `[`number`](https://www.twicpics.com/docs/api/manipulations/?utm_source=nuxt&utm_medium=organic&utm_campaign=provider#number)`|'flip'|'left'|'right' }`
+
+```html
+<NuxtImg
+  provider="twicpics"
+  src="/football.jpg"
+  :modifiers="{turn:'left'}" <!-- turns image at -90° -->
+/>
+
+<NuxtImg
+  provider="twicpics"
+  src="/football.jpg"
+  :modifiers="{turn:180}" <!-- turns image at 180° -->
+/>
+```
+
+More informations [about `turn` here](https://www.twicpics.com/docs/api/transformations/?utm_source=nuxt&utm_medium=organic&utm_campaign=provider#turn).
+
+### zoom
+
+Zooms into the image by a factor equal or superior to 1 towards the focus point while preserving the image size.
+
+Syntax: `{ zoom: `[`number`](https://www.twicpics.com/docs/api/manipulations/?utm_source=nuxt&utm_medium=organic&utm_campaign=provider#number)` }`
+
+```html
+<NuxtImg
+  provider="twicpics"
+  src="/components/flip/cherry-3.jpg"
```

Comment:
```suggestion
  src="/cherry-3.jpg"
```

Do you mind also removing the `/components/flip` part on other examples?

It is not necessary.